src/vm.ts

```diff
@@ -24,14 +24,51 @@ const dstColumn: String = columns[i];
 i++;
 const executionColumn: String = columns[i];
 i++;
+
+type builtins = {
+  output: string;
+  pedersen: string;
+  range_check: string;
+  ecdsa: string;
+  bitwise: string;
+  ec_op: string;
+  keccak: string;
+  poseidon: string;
+};
+
+const builtins = {
+  output: null,
+  pedersen: null,
+  range_check: null,
+  ecdsa: null,
+  bitwise: null,
+  ec_op: null,
+  keccak: null,
+  poseidon: null,
+};
+
 const program: any[][] = programSheet.getRange("A2:A").getValues();
 
+function initialize_builtins(): void {
+  const keys: string[] = Object.keys(builtins);
+  for (var key of keys) {
```

Comment:
I'd rather enumerate the keys, use a local counter starting at 0, and offset with `executionColumn` to make it clearer that the builtins segments are put after the execution one

Reply:
Thanks for the pointer! I pushed a change, and now it uses offset from `executionColumn` instead. Also, please feel free to suggest any other change even related to refactoring to reduce code smells. I am not too familiar with JavaScript, so there might be some. Therefore, your input will always be welcome :D